src/Symfony/Component/Console/Tests/Fixtures/application_1.json

```diff
@@ -104,8 +105,9 @@
         },
         {
             "name": "list",
+            "hidden": false,
```

Comment:
Maybe do a mixed test? One hidden, one not. While avoiding lots of extra boiler plate fixtures.